app/controllers/spree/admin/reports_controller_decorator.rb

```diff
@@ -244,7 +244,9 @@ def users_and_enterprises
   def xero_invoices
     if request.get?
       params[:q] ||= {}
-      params[:q][:completed_at_gt] = Time.zone.now.beginning_of_month
+      params[:q][:completed_at_gt] = Time.zone.today.beginning_of_month
+      params[:invoice_date] = Time.zone.today
+      params[:due_date] = Time.zone.today + 1.month
```

Comment:
Not really ... perhaps we can sit together tomorrow for a bit. If it takes too much time, let's not have it block this PR.